js/evlPlayer.js

```diff
@@ -27,7 +27,7 @@
 					content = "There was an error while loading this video";
 				}
 				if (data.notice) {
-					content = content + "<small>" + data.notice + "</small>";
+					content = content + "<small>" + encodeURIComponent(data.notice) + "</small>";
```

Comment:
Note from refinement: @latata suggested using `text` property instead of concatenating and using `encodeURIComponent()`

Reply:
`encodeURIComponent` theoretically should be sufficient to fix XSS vulnerability but this function was not designed for that purpose. Using jquery's `text()` (or native `.textContent`) sounds like a better solution.

Reply:
Do you mean something like below? I believe it's better, because special characters aren't being encoded as well, but out of curiosity, is there any other reason that this would be preferable approach?
```
if (data.notice) {
	const noticeText = $("<small>").text(data.notice);
	content += noticeText.prop('outerHTML');
}
```

Reply:
I think the reason you provided (special chars) is good enough to use `text()` instead of `encodeURIComponent` ;)